R/test-files.R

```diff
@@ -126,6 +126,7 @@ test_file <- function(path, reporter = default_reporter(), env = test_env(),
     reporter = reporter,
     start_end_reporter = start_end_reporter,
     {
+      get_reporter()$start_file(basename(path))
```

Comment:
We need to do this separately from the lister, because the lister is not part of the default reporter.

Reply:
this would make a good comment in the source imo